### PR TITLE
Increase API version

### PR DESCRIPTION
**Pull request by vbauzysvmware**
Due vCD 9.1 support ends we changing API to minimum vCD 9.5 version.

* Increased API version to 31.0
* Improved catalog test
* Added fields AdminVdc.UniversalNetworkPoolReference, VM.Media`

Testes Verified with 9.5 and 9.7 and with terraform tests on 9.5 and 9.7

Note: terraform need change to address:
```
    testing.go:654: Step 0 error: Check failed: 2 errors occurred:
        	* Check 7/52 error: vcd_edgegateway.egw: Attribute 'fips_mode_enabled' found when not expected
        	* Check 34/52 error: data.vcd_edgegateway.egw: Attribute 'fips_mode_enabled' found when not expected
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 * Added methods `OrgVdcNetwork.Update`, `OrgVdcNetwork.UpdateAsync`, and `OrgVdcNetwork.Rename` [#292](https://github.com/vmware/go-vcloud-director/pull/292)
 * Added methods `EdgeGateway.Update` and `EdgeGateway.UpdateAsync` [#292](https://github.com/vmware/go-vcloud-director/pull/292)
-
+* Increment vCD API version used from 29.0 to 31.0
+    * Add fields `AdminVdc.UniversalNetworkPoolReference, VM.Media`
```
Comment (Didainius) [inline CHANGELOG.md:6]: Probably some morning typo

File: govcd/api_vcd.go [hunk 2]
```diff
@@ -89,7 +89,7 @@ func NewVCDClient(vcdEndpoint url.URL, insecure bool, options ...VCDClientOption
 	// Setting defaults
 	vcdClient := &VCDClient{
 		Client: Client{
-			APIVersion: "29.0", // supported by vCD 9.0, 9.1, 9.5, 9.7, 10.0
+			APIVersion: "31.0", // supported by vCD 9.0, 9.1, 9.5, 9.7, 10.0
```
Comment (Didainius) [inline govcd/api_vcd.go:92]: Comment needs fixing :)

Comment (vbauzysvmware) [reply govcd/api_vcd.go]: removed and added versions

File: CHANGELOG.md (hunk 1 above)
Comment (vbauzysvmware) [reply CHANGELOG.md]: 6th line is additional info for 5th line

**Pull request merged**